app/assets/javascripts/directives/clear_field_set_focus.js

```diff
@@ -2,24 +2,31 @@ ManageIQ.angular.app.directive('clearFieldSetFocus', ['$timeout', 'miqService',
   return {
     require: 'ngModel',
     link: function (scope, elem, attr, ctrl) {
+      var model = function() {
+        if (scope.$parent.angularForm) {
+          return scope.$parent.$eval(scope.$parent.angularForm.model || scope.$parent.model);
+        } else {
+          return scope.$eval(scope.angularForm.model || scope.model);
+        }
+      };
       scope['form_passwordfocus_' + ctrl.$name] = elem[0];
 
       var option = attr.clearFieldSetFocus;
 
-      scope.$watch('bChangeStoredPassword', function(value) {
+      scope.$watch('vm.bChangeStoredPassword', function(value) {
```

Comment:
Are all controllers using `_auth_credentials_angular_bootstrap` already `controllerAs`? (Otherwise this won't work there..)